### PR TITLE
Create Swiftplot_Docker_setup.md

### PR DESCRIPTION
**Pull request by anigasan**
Created instruction notes to setup Docker instance for swift-jupyter

### CODE REVIEW

File: Swiftplot_Docker_setup.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
+# Setting up docker container for SwiftPlot and Swift-Jupyter
+
+To use Swiftplot in a notebook, it requires setting up of swift-jupyter environment, either in Ubuntu or Docker. While either of the approaches are not very difficult, Docker approach provides a consistent way to setup environment. In this document, I will highlight steps needed to setup docker container for Swiftplot.
```
Comment (KarthikRIyer) [inline Swiftplot_Docker_setup.md]: `This document highlights the steps needed to setup docker container for Swiftplot.`
Documentations shouldn't be in 1st person, preferably.

File: Swiftplot_Docker_setup.md [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+
+<H3> Install Docker Desktop </H3>
+
+1. First step is to download Docker desktop application from www.docker.com. To download your OS specific version of desktop application, please singup (or signin if you have signed earlier). 
+2. Once successfully signed-in, head out to download area and choose right version (based on your OS) of Dcoker to download.
```
Comment (KarthikRIyer) [inline Swiftplot_Docker_setup.md]: Typo: Use Docker instead of Dcoker.

**Review by KarthikRIyer — CHANGES_REQUESTED** [review]

**Comment by KarthikRIyer** [comment]
I suppose this will be useful for people using Windows or mac. So could you add a section in the README that directs people to this file?

File: Swiftplot_Docker_setup.md (hunk 1 above)
Comment (anigasan) [reply Swiftplot_Docker_setup.md]: @KarthikRIyer  - It is a very good point, especially setup documentation!! I will make changes in related areas.

File: Swiftplot_Docker_setup.md (hunk 2 above)
Comment (anigasan) [reply Swiftplot_Docker_setup.md]: Thanks for pointing it out, late-night work :) I will rectify it now.

**Comment by KarthikRIyer** [comment]
Thanks for the update!

**Pull request merged**